robot/Cumulus/resources/ObjectMangerPageObject.py

```diff
@@ -20,6 +20,62 @@ def open_fields_and_relationships(self, object_name):
 		search_button = npsp_lex_locators['object_manager']['input'].format("globalQuickfind")
 		self.salesforce.wait_until_loading_is_complete()
 		self.selenium.wait_until_page_contains_element(search_button)
+		if object_name == "Payment":  # Need this extra logic as Payment is not the same as other standard objects
+			self.selenium.get_webelement(search_button).send_keys(object_name)
+			self.selenium.get_webelement(search_button).send_keys(Keys.ENTER)
+			object = npsp_lex_locators['object_manager']['object_result'].format(object_name)
```

Comment:
I think u could just navigate to this url 'lightning/setup/ObjectManager/home 'and perform all the steps to go to fields and relationships as this problem is common for all objects(not just payments) that are not the salesforce standard ones

Reply:
addressed this